### DISCUSSION TITLE
mongoose 사용하는 것이 어떨까요?

### DISCUSSION THREAD

**Discussion by ghost** [tastekim/WeAllLie-BE#9] [Ideas]
저희가 ORM 없이 mongoDB 사용했던 게 파이썬 사용할 때였는데...
파이썬에서는 딱히 ORM 없이도 크게 복잡하지 않게 몽고디비 사용했던것 같긴 해요.
근데 Node.js에서 어떻게 사용해야 하나 검색해보니까... 조금 복잡하더라구요...
연결된 커넥션 함수 안에서 디비를 호출하고 이런 식이라 한 번 디비 호출하려고 해도 코드가 너무 길어지더라구요.
그냥 사전 작업 (스키마/모델 생성) 이 좀 필요해도 파일 분리해두면 되니까 필요한 곳에서는 require 한 번 하고 바로 디비에 접근하는게 
보기 좋을 것 같고 사용하기에도 편할 것 같습니다.
어떠신가요 다들?

**Comment by tastekim** [comment]
그럼 이따가 DB 설계부분 모여서 얘기해보는건 어떠세요 ?
@Minhyeong-S @kimhsno1 @yunjin5450

**Reply by ghost** [reply]
넵!!

**Reply by yunjin5450** [reply]
넵 좋아용

**Reply by ghost** [reply]
유저 정보에 어떤 항목들을 넣을 건지도 확실하게 정하면 좋을 것 같아요!
제가 임의로 하기 그런 부분 같아서..

**Reply by tastekim** [reply]
10시에 각자 생각 좀 해본 후에 모여서 논의해봅시다 :) @Minhyeong-S @yunjin5450 @kimhsno1

**Reply by yunjin5450** [reply]
하잇~~
